src/sentry/tasks/merge.py

```diff
@@ -257,11 +257,16 @@ def merge_objects(models, group, new_group, limit=1000,
                             )
                     elif model == GroupTagValue:
                         with transaction.atomic(using=router.db_for_write(model)):
-                            model.objects.filter(
+                            new_obj = model.objects.get(
```

Comment:
I don't like this name (it's not new) but this follows convention established elsewhere in this function.